pom.xml

```diff
@@ -24,8 +24,8 @@
 
     <distributionManagement>
         <repository>
-            <id>bintray-datadog-api-client</id>
-            <url>https://api.bintray.com/maven/${bintray.repo}/${bintray.package}/</url>
+            <id>nexus</id>
+            <url>http://localhost:8081/nexus/content/repositories/snapshots</url>
```

Comment:
❓

Reply:
I'll definitely change this before merging, just a local sonatype repository for testing 🙂 
Right now to test things out I'm using - https://hub.docker.com/r/sonatype/nexus